sql/core/src/test/scala/org/apache/spark/sql/parquet/ParquetSchemaSuite.scala

```diff
@@ -467,7 +467,7 @@ class ParquetSchemaSuite extends ParquetSchemaTest {
     """message root {
       |  optional group f1 (LIST) {
       |    repeated group list {
-      |      optional int32 element;
+      |      optional int32 array;
```

Comment:
Changes made are limited to Catalyst to Parquet schema conversion, so there's no need to modify this Parquet to Catalyst test case.